stdlib/private/StdlibUnittest/RaceTest.swift

```diff
@@ -55,6 +55,8 @@ func autoreleasepool(invoking code: () -> Void) {
 }
 #endif
 
+// FIXME: On FreeBSD, code in this file doesn't compile
```

Comment:
This conditional removal of code was due to the pthread wrappers not working in a previous commit. The code is reinstated now since the wrappers got fixed due to @tinysun212 'a Cygwin patch work